monitoring/dashboard.py

```diff
@@ -630,6 +819,18 @@ def top10_errors_by_bucket(title, code):
                 top10Error500ByBucket,
                 top10Error5xxByBucket
             ], height=8),
+            RowPanel(title="Quotas"),
+            layout.row([
+                layout.resize([quotaHealth], width=6),
+                layout.resize([quotaStatusCode], width=6),
+                quotaByAction,
+            ], height=8),
+            layout.row([
+                layout.resize([bucketQuotaCounter], width=3),
+                layout.resize([accountQuotaCounter], width=3),
+                layout.resize([averageMetricsRetrievalLatencies], width=6),
+                averageQuotaLatencies,
```

Comment:
we store histogram metrics, but only display the average → we should try to use the information to provide more info, like the distribution (heatmap) or 95th percentile...

Reply:
What do you think of the heatmap I added? (see comment above)